src/mixer/state.ts

```diff
@@ -454,7 +465,18 @@ export const load = (
     const blob = new Blob([rawData]);
     const objectUrl = URL.createObjectURL(blob);
 
-    const channelOutputId = getState().settings.channelOutputIds[player];
+    let channelOutputId = getState().settings.channelOutputIds[player];
+    // If the player setting doesn't exist, reset the settings. (Happens after player count is increased)
+    if (!channelOutputId) {
+      channelOutputId = INTERNAL_OUTPUT_ID;
+      dispatch(
+        changeSetting({
+          key: "channelOutputIds",
+          // @ts-ignore
+          val: Array(PLAYER_COUNT).fill(channelOutputId),
```

Comment:
Ooer

Reply:
Is this not the same as in python when you do [Value]*2 or whatever? If it's about the @ts-ignore, that was already there, will remove if it's not necessary.

Reply:
Yeah, I’m not a fan of unnecessary ts-ignores